lib/cocoapods/generator/prepare_artifacts_script.rb

```diff
@@ -59,113 +59,103 @@ def generate
       #
       def script
         script = <<-SH.strip_heredoc
-          #!/bin/sh
```

Comment:
just updated indentation. I think because I added string interpolation to the mix it broke indentation. Either way no changes here except using the script constants instead.